src/components/data_submission/nih_institutions.js

```diff
@@ -0,0 +1,29 @@
+export const nihInstitutions = [
```

Comment:
This comment is more of an architectural question/suggestion. We have developed a schema to define all of these values already in Consent. It seems duplicative to define these here as well. What I think I would prefer to see is something that queries for the schema, reads it, and uses what's in the schema as opposed to doing all of that work twice. 

Caveat - the one thing that's missing right now is the ability for duos to query for the actual schema. I'm pretty sure we don't have a story for that. I'll get that ball rolling.